systemtests/src/test/java/io/enmasse/systemtest/brokered/SmokeTest.java

```diff
@@ -99,16 +98,20 @@ void testCreateDeleteAddressSpace() throws Exception {
         QueueTest.runQueueTest(amqpQueueCliC, queueB);
     }
 
-    @Test()
-    @Disabled("disabled until #346 will be fixed")
+    @Test
     void testCreateAlreadyExistingAddress() throws Exception {
         AddressSpace addressSpaceA = new AddressSpace("brokered-a", AddressSpaceType.BROKERED, AuthService.STANDARD);
         createAddressSpace(addressSpaceA);
         Destination queueA = Destination.queue("brokeredQueueA", getDefaultPlan(AddressType.QUEUE));
         setAddresses(addressSpaceA, queueA);
 
-        Destination topicA = Destination.topic("brokeredTopicA", getDefaultPlan(AddressType.TOPIC));
-        assertThrows(AddressAlreadyExistsException.class, () -> setAddresses(addressSpaceA, topicA),
-                "setAddresses does not throw right exception");
+        Destination topicA = Destination.topic("brokeredQueueA", getDefaultPlan(AddressType.TOPIC));
+        try {
+            setAddresses(addressSpaceA, topicA);
+        } catch (Exception e) {
+            if (e.getMessage().contains("\"status\":\"Failure\",\"code\":409,\"reason\":\"AlreadyExists\"")) {
+                ITestSeparator.log.info("Test successful, address already exists and is not modified");
```

Comment:
why you call itest separator, logging is called automatically

Reply:
removed